### PR TITLE
Moved test out of run_in_eager_and_graph_mode in resampler_ops.

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**

### CODE REVIEW

File: tensorflow_addons/image/resampler_ops_test.py [hunk 1]
```diff
@@ -193,62 +193,64 @@ def _test_op_backward_pass(self, on_gpu, dtype):
...
-        warp_shape = (batch_size, warp_height, warp_width, 3)
-        warp = np.zeros(warp_shape)
-
-        with self.assertRaisesRegexp(
-            tf.errors.UnimplementedError,
-            "Only bilinear interpolation is supported, warping",
-        ):
-            self.evaluate(resampler_ops.resampler(data, warp))
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_op_errors():
```
Comment (autoih) [inline tensorflow_addons/image/resampler_ops_test.py]: LGTM. Thanks, @gabrieldemarmiesse. 
Just curious about how we decide what tests should be moved out? Could you please also share the thoughts?  The [PR](https://github.com/tensorflow/addons/pull/1438/files) is a good example of my question: can we also move this? https://github.com/tensorflow/addons/blob/81479e2fcfee09541dae57ccfd9c4e43fe0ff311/tensorflow_addons/layers/normalizations_test.py#L292

Comment (gabrieldemarmiesse) [reply tensorflow_addons/image/resampler_ops_test.py]: Sorry I forgot to link the issue: #1328

We should move all the tests, including the one you're mentioning. But I'm doing it one at a time to make pull requests easier to review. You wouldn't want to review 2000+ lines of code at once :)

Comment (autoih) [reply tensorflow_addons/image/resampler_ops_test.py]: That makes sense. Thanks @gabrieldemarmiesse.

**Comment by autoih** [comment]
@gabrieldemarmiesse, not sure why it's waiting for `cla`, can you please help check? Thanks.

**Comment by gabrieldemarmiesse** [comment]
The cla is not preventing us from merging. So no worries on this part.

**Comment by autoih** [comment]
It seems there are conflicts. Could you fix them? Thanks, @gabrieldemarmiesse.

**Pull request closed without merging**